### PR TITLE
Set default softmax axis=-1

### PR DESCRIPTION
**Pull request by jiafatom**
https://keras.io/activations/, keras.activations.softmax(x, axis=-1) 
-1 should be default.

### CODE REVIEW

File: keras2onnx/common/onnx_ops.py [hunk 1]
```diff
@@ -405,7 +405,7 @@ def apply_identity(scope, input_name, output_name, container, operator_name=None
     container.add_node('Identity', input_name, output_name, name=name)
 
 
-def apply_softmax(scope, input_name, output_name, container, operator_name=None, axis=1):
+def apply_softmax(scope, input_name, output_name, container, operator_name=None, axis=-1):
```
Comment (wenbingl) [inline keras2onnx/common/onnx_ops.py]: Are you sure -1 is also supported by ONNX?

Comment (jiafatom) [reply keras2onnx/common/onnx_ops.py]: will do this in conv.py
